src/spirc.rs

```diff
@@ -246,7 +295,9 @@ impl SpircTask {
 
         if frame.get_ident() == self.ident ||
            (frame.get_recipient().len() > 0 && !frame.get_recipient().contains(&self.ident)) {
-            return;
+            if frame.get_typ() == MessageType::kMessageTypeNotify {
```

Comment:
You only need to broadcast the `kMessageTypeNotify` using `self.notify(None);`, not the `kMessageTypePause`.